docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+As you can see in the file, there are quite a number of parameters you either have the option to set or will need to set for your node to work properly. In general, if you do not have the need to change a tuning value, leave it alone. You will, however, likely need to adjust the various addresses, specify the database type you want to use, as well as to specify where the MSP for the node is located.
+
+You have two main options for tuning your configuration.
+
+1. Edit the YAML file bundled with the binaries.
+2. Use environment variable overrides when deploying (if using Docker).
+
+The former option has the advantage of persisting your changes whenever you bring down and bring back up the node. The downside is that you will have to port the options you customized to the new YAML when upgrading to a new binary version (you should use the latest YAML when upgrading to a new version). By using a file like `./env<name of node>.list` that contains your environment variables, it is easy to port your changes to a new YAML.
+
+Either way, here are some values in `core.yaml` you might want to edit.
+
+* `localMspID`: this is the name of the local MSP, generated by your CA, of your peer organization. This is where your peer organization admins will be listed.
+
+* `mspConfigPath`: the place where the local MSP for the peer is located. Note that it is a best practice to mount this volume external to your container.
+
+* `peer/address`: represents the endpoint to other peers in the same organization, an important consideration when establishing gossip communication within an organization.
+
+* `# TLS enabled`, `Enabled: true`: This is where you specify whether you want to [enable TLS](enable_tls.html). If you set this value to `true`, you will have to specify the locations of the relevant TLS certificates. Note that all of the nodes in a network (both the peers and the ordering nodes, must either all have TLS enabled or not enable. For production networks, it is highly recommended to enable TLS). As with your MSP, it is a best practice to mount this volume external to your container.
+
+* `ledger`: users have a number of decisions to make about their ledger, including the state database type (level or CouchDB), and its location (specified in `fileSystemPath` --- not that for CouchDB in particular that it is a best practice to operate your state database external to the peer).
+
+* `gossip`: there are a number of configuration options to think about when setting up [gossip](gossip.html), including the `externalEndpoint` (which makes peers discoverable to peers owned by other organizations) as well as the `bootstrap` address (which identifies a peer in the peer's own organization).
```

Comment:
The external endpoint setting is so important, I wonder if "might want to edit" doesn't do it justice. Perhaps "you might want to edit" would be better as "you must review", so the casual reader doesn't skip over them thinking the defaults are OK?